nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/themes/nifi.scss

```diff
@@ -49,355 +53,100 @@ $material-primary-light-palette: (
-        A100: #ffffff,
-        A200: #ffffff,
-        A400: rgba(black, 0.87),
-        A700: rgba(black, 0.87),
-    )
-);
-
-// The $nifi-canvas-light-palette defines the ACCENT palette for all flow designer canvas components that make up the NiFi canvas theme used throughout Apache NiFi
-$nifi-canvas-accent-light-palette: (
-    // 50 -> 900 are the PRIMARY colors defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors for primary color #52bf7e
-    50: #e6f6ec,
-    100: #c3e8d0, // "lighter" hue for this palette.
-    200: #9dd9b2, //.running
-    300: #73ca94, //.backpressure-percent
-    400: #52bf7e, // color="primary" Default hue for this palette. Also .up-to-date
-    500: #2cb367,
-    600: #1A9964, //.version-control
-    700: #016131, // "darker" hue for this palette Also .bulletin-normal
-    800: #0000ff, //.endpoint, g.process-group.drop rect.border
-    900: #00ff00, //.connectable-destination, .connector.connectable
-
-    // A100 - A700 are the ANALOGOUS colors but are more customized. These colors are used to highlight, warn, denote midpoints and labelpoints, etc
-    A100: #ffef85, //.selected
-    A200: #f8bf47, //.invalid, .is-missing-port, circle.context
-    A400: #bda500, //.backpressure-percent-warning, .bulletin-warn, .backpressure-percent.warning, text.run-status-icon
-    A700: #ffcc00, //g.connection.selected rect.border, .connection-selection-path, .midpoint, .labelpoint
-
-    // These are the $nifi-canvas-accent-light-palette PRIMARY palette contrast colors. These color do not really get defined by https://m2.material.io/design/color/the-color-system.html#tools-for-picking-colors.
-    // Instead if we look to the Angular Material provided palettes we see that these fields are typically rgba(black, 0.87) or white. These values are particularly important
-    // for light mode and dark mode as these values set the colors for the text when displayed against the primary background on a button, badge, chip, etc.
-    //
-    // NOTE: Care should be taken here to ensure the values meet accessibility standards.
-    //
-    // NOTE: When creating the Material palette definition mat.define-palette($nifi-canvas-accent-light-palette, 400, 100, 700);
-    // Since 400 is the default the contrast-400 will be used as the default text color in some cases.
-    contrast: (
-        50: rgba(black, 0.87),
-        100: rgba(black, 0.87),
-        200: rgba(black, 0.87),
-        300: rgba(black, 0.87),
-        400: rgba(black, 0.87),
-        500: rgba(black, 0.87),
-        600: #ffffff,
-        700: #ffffff,
-        800: #ffffff,
-        900: #ffffff,
-        A100: rgba(black, 0.87),
-        A200: rgba(black, 0.87),
-        A400: rgba(black, 0.87),
-        A700: rgba(black, 0.87),
+        50: $on-surface-dark,
+        100: $on-surface-dark,
+        200: $on-surface-dark,
+        300: $on-surface-dark, 
+        400: $on-surface-dark,
+        500: $on-surface-dark,
```

Comment:
```suggestion
        500: $on-surface-light,
```

Reply:
That would be #728e9b and #ffffff as contrast colors. [That only gets a 3.46:1 color contrast ratio](https://webaim.org/resources/contrastchecker/?fcolor=728E9B&bcolor=FFFFFF) 

[$on-surface-dark gets a contrast ratio of 5.42:1 ](https://webaim.org/resources/contrastchecker/?fcolor=000000de&bcolor=728E9B)

Reply:
I am not totally following what you mean. I simply looked at the palettes provided by Angular Material (I assume those are all tested with Angular Material components and that the contrast ratios are acceptable). The nifi `$material-primary-light-palette` is very close to the Angular Material `$blue-grey-palette`:

![Screenshot 2024-03-13 at 1 51 02 PM](https://github.com/apache/nifi/assets/6797571/eb4650fc-23c1-4889-ab0b-0eb9b98c4e86)

The contrast colors in the `$blue-grey-palette` use a light color for contrast hues 400-900 and A400 and A700. What I am requesting is that the `$material-primary-light-palette` should be extremely similar.

If I run this th
![Screenshot 2024-03-13 at 2 01 47 PM](https://github.com/apache/nifi/assets/6797571/a9919118-0d10-40a5-b375-192ce3c053a4)
![Screenshot 2024-03-13 at 2 02 09 PM](https://github.com/apache/nifi/assets/6797571/7a502779-bad8-42e2-8fec-4af5cb5bba51)
![Screenshot 2024-03-13 at 2 02 18 PM](https://github.com/apache/nifi/assets/6797571/4a97b9cf-6a52-4220-bdc5-87993d6f13e9)
![Screenshot 2024-03-13 at 2 02 33 PM](https://github.com/apache/nifi/assets/6797571/3a941f19-589b-43fc-92b0-c753f12fa386)
![Screenshot 2024-03-13 at 2 02 49 PM](https://github.com/apache/nifi/assets/6797571/62e1e279-e598-401a-a952-9052bb5eaf65)
![Screenshot 2024-03-13 at 2 02 57 PM](https://github.com/apache/nifi/assets/6797571/05543201-7239-4e13-8acb-929118987dd8)
eme with the Angular Material demo application and view components they are not looking correct:

Reply:
looks good now